micrometer-core/build.gradle

```diff
@@ -283,6 +283,11 @@ java11Test {
 }
 
 compileJava11TestJava {
+    options.errorprone.enabled = false
     sourceCompatibility = JavaVersion.VERSION_11
     targetCompatibility = JavaVersion.VERSION_11
 }
+
+compileJava11Java {
+    options.errorprone.enabled = false
```

Comment:
Disables errorprone for the multi-release JAR source. There was an error trying to compile it, all of the classes in the source are deprecated and duplicated in micrometer-java11, and errorprone runs on the micrometer-java11 module. So I don't think there's any point trying to get errorprone to work here at this time.

Reply:
I guess it's the same issue as the JDK 11 build, since the multi-release JAR plugin will download JDK 11 even if we're using a JDK that could compile to 11 via the `--release` flag.